src/platform/user/profile/actions/mhv.js

```diff
@@ -17,3 +20,35 @@ export function fetchMHVAccount() {
       );
   };
 }
+
+/**
+ * Get message signature from user preferences.
+ * @returns {Object} signature object {data: {signatureName, includeSignature, signatureTitle}, errors:{}, metadata: {}}
+ */
+export const getMessagingSignature = () => {
+  return async dispatch => {
+    try {
+      const response = await apiRequest(
+        `${myHealthApiBasePath}/messaging/messages/signature`,
```

Comment:
looks like you are having this already declared in constants file. Would it make sense to use it from `src/platform/user/profile/vap-svc/constants/index.js` ?